src/gemdat/transitions.py

```diff
@@ -405,3 +406,197 @@ def _calculate_occupancy(atom_sites: np.ndarray) -> dict[int, int]:
     occupancy = dict(zip(unq, counts))
     occupancy.pop(NOSITE, None)
     return occupancy
+
+
+def _wrap_pbc(coord: np.ndarray, size: tuple) -> tuple:
```

Comment:
I think this should return a numpy array.

Reply:
To be a node or an edge of a graph object it needs to be hashable, but I can make it a tuple outside and leave this with a more general np.ndarray

Reply:
Yeah, that makes sense. I saw a bunch of type conversions from array<>tuple, which is what triggered this comment. Either way is okay.